### PR TITLE
add Dutch translation

### PR DESCRIPTION
**Pull request by threes-was-taken**

### CODE REVIEW

**Comment by jtoar** [comment]
Thanks @threes-was-taken! @noire-munich ok if I tag you on this one?

**Comment by noire-munich** [comment]
@jtoar sure!

File: web/src/locales/nl.json [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      "title": "About"
+    },
+    "community": {
+      "code": "Code of Conduct",
+      "contributors": "Contributors Meetup",
+      "join": "Word lid van de community",
+      "makers": "Makers Hours",
+      "title": "Community"
+    },
+    "misc": {
```
Comment (noire-munich) [inline web/src/locales/nl.json:127]: I've fixed the commits, `misc` doesn't exist anymore.

File: web/src/locales/nl.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      "shop": "Shop",
+      "title": "Andere"
+    },
+    "newsletter": {
+      "button": "Subscribe",
+      "subtitle": "Blijf op de hoogte van wat er gebeurt met RedwoodJS!",
+      "title": "Abonneer op onze nieuwsbrief"
+    },
+    "social": {
```
Comment (noire-munich) [inline web/src/locales/nl.json:136]: I've fixed the commits, `misc` doesn't exist anymore. It was replaced by `announcements`.

**Review by noire-munich — CHANGES_REQUESTED** [review]
@threes-was-taken there are two changes from conflict solving, if you can have a look.
Work on the original PR is not complete yet, so we might bounce a bit more.

File: web/src/locales/nl.json (hunk 2 above)
Comment (threes-was-taken) [reply web/src/locales/nl.json:136]: Do I remove the 'misc' key from the json or do I rename the key to ```announcements``` ?

Comment (noire-munich) [reply web/src/locales/nl.json]: What you did is good :+1:

**Pull request closed without merging**